solarforecastarbiter/validation/quality_mapping.py

```diff
@@ -56,7 +56,13 @@
 # flags that should typically be discarded before resampling because they
 # represent truly bad data
 DISCARD_BEFORE_RESAMPLE = [
-    'USER FLAGGED', 'LIMITS EXCEEDED', 'INCONSISTENT IRRADIANCE COMPONENTS'
+    'USER FLAGGED',
+    'LIMITS EXCEEDED',
+    'INCONSISTENT IRRADIANCE COMPONENTS',
+    'STALE VALUES',
+    'INTERPOLATED VALUES',
+    'DAYTIME STALE VALUES',
+    'DAYTIME INTERPOLATED VALUES'
```

Comment:
Redundant to add the daytime stale/interpolated https://github.com/SolarArbiter/solarforecastarbiter-core/blob/85c1e6d887e0414c96754f8014df6a0194c4103b/solarforecastarbiter/validation/quality_mapping.py#L49-L54

Reply:
This isn't defining the flags, it's just saying they should be treated this way by default. It's currently only used by the dashboard:

https://github.com/SolarArbiter/solarforecastarbiter-dashboard/blob/0ffb40d9f15c1b457eb16bb321438703084a5777/sfa_dash/form_utils/converters.py#L542-L555